vector/src/main/scala/geotrellis/vector/package.scala

```diff
@@ -28,13 +28,13 @@ package object vector extends SeqMethods
     with triangulation.Implicits
     with voronoi.Implicits {
 
-  type PointFeature[D] = Feature[Point, D]
-  type LineFeature[D] = Feature[Line, D]
-  type PolygonFeature[D] = Feature[Polygon, D]
-  type MultiPointFeature[D] = Feature[MultiPoint, D]
-  type MultiLineFeature[D] = Feature[MultiLine, D]
-  type MultiPolygonFeature[D] = Feature[MultiPolygon, D]
-  type GeometryCollectionFeature[D] = Feature[GeometryCollection, D]
+  type PointFeature[+D] = Feature[Point, D]
```

Comment:
@pomadchin Didn't see this hidden in here. This change actually makes sense. Curious what problem inspired it.

Reply:
@echeipesh I think it’s 2.11 compiler bug / feature. The only change that needs to be done to support 2.12; mb it makes sense to create a separate PR with this changes in order to include it into 2.0